doc/filters.texi

```diff
@@ -20769,6 +20769,36 @@ Convert HDR(PQ/HLG) video to bt2020-transfer-characteristic p010 format using li
 @end example
 @end itemize
 
+@section tonemap_vappi
+
+Perform HDR(High Dynamic Range) to SDR(Standard Dynamic Range) conversion with tone-mapping.
+It maps the dynamic range of HDR10 content to the SDR content.
+It only accepts HDR10 as input temporarily.
+
+It accepts the following parameters:
+
+@table @option
+@item type
+Specify the tone-mapping operator to be used.
```

Comment:
I would like you to remove the 'type' option. it is easy to derive it from the color_transfer. if the output is smpte2084 or HLG, you can easily say this is H2H, otherwise H2S. I still think tone-mapping to 10bit SDR is important, please teach me if I am wrong. so please help discuss with media-driver developer to add 10bit SDR support.

Reply:
I agree with you. But I think it is more clear for user to use this filter if keeping this option. How about keeping the 'type' option in this version?

Reply:
I didn't see 'more clear'. this is redundant with output transfer function. take a look at other tone-mapping filters in FFmpeg.

Reply:
Fixed in the new version.